store/sqlstore/upgrade.go

```diff
@@ -514,3 +515,16 @@ func UpgradeDatabaseToVersion55(sqlStore SqlStore) {
 	// 	saveSchemaVersion(sqlStore, VERSION_5_5_0)
 	// }
 }
+
+func UpgradeDatabaseToVersion56(sqlStore SqlStore) {
+	// TODO: Uncomment following condition when version 5.6.0 is released
+	//if shouldPerformUpgrade(sqlStore, VERSION_5_5_0, VERSION_5_6_0) {
+
+	// migrating user's accepted terms of service data into the new table
+	if sqlStore.DoesColumnExist("Users", "AcceptedTermsOfServiceId") {
+		sqlStore.GetMaster().Exec("INSERT INTO UserTermsOfService SELECT Id, AcceptedTermsOfServiceId as TermsOfServiceId, :CreateAt FROM Users WHERE AcceptedTermsOfServiceId != \"\" AND AcceptedTermsOfServiceId IS NOT NULL", map[string]interface{}{"CreateAt": model.GetMillis()})
```

Comment:
Can you make sure this is acceptably fast on a DB with 60000 users?

Reply:
I ran the query directly in the database and it took less than a second to run for a dummy dataset of 60000 users. Its a simple query for no joins and stuff, so it shouldn't be slow.